cypress.config.ts

```diff
@@ -0,0 +1,30 @@
+import { defineConfig } from 'cypress'
+
+export default defineConfig({
+    e2e: {
+        baseUrl: 'https://deriv.com',
+    },
+    env: {
+        email: 'test@example.com',
+        baseUrl: 'https://deriv.com',
+        base_url_eu: 'https://eu.deriv.com',
```

Comment:
I just found out we can just test either eu or non-eu url, as for all users it will be redirected anyway. 
```suggestion
```

Reply:
@habib-deriv Then one test case will always fail? how we will handle it? 🤔